src/app/base/components/dropdown/dropdown.component.ts

```diff
@@ -0,0 +1,181 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
```

Comment:
as mentioned before, this feels very hard to maintain being completely hardcoded. i think to reduce our stress in the long run we should aim for a more flexible solution (like with the dynamic dropdown)